person.rb

```diff
@@ -1,22 +1,23 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rentals'
 
-# Person class provides the default implementation of the correct_name method
+# Person class provide the defuslt implementation of the correct_name method
 class Person < Nameable
   def initialize(age, name = 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
-
-  # Getter and setter in one
+  # getter and setter in one
   attr_accessor :name, :age
 
```

Comment:
- [ ] Please make sure the rentals variable is readable and writable to work for the associations as expected,  To achieve this, You can add that in the accessor.